index.js

```diff
@@ -0,0 +1,30 @@
+const taskInput = document.getElementById('myInput');
+const taskList = document.getElementById('taskList');
+
+taskInput.addEventListener('keydown', function (event) {
+  const taskText = taskInput.value.trim();
+  if (event.key === 'Enter' && taskText !== '') {
+    const taskItem = createTaskItem(taskText);
+    taskList.appendChild(taskItem);
+    taskInput.value = '';
+  }
+});
```

Comment:
하는일이 다른 문끼리는 개행을 해주는게 좋을 것 같아요.
개행도 코드의 일부분이고 가독성을 향상 시켜줄 수 있어요.

```suggestion
  const taskText = taskInput.value.trim(); // 데이터 준비
  
  if (event.key === 'Enter' && taskText !== '') {
    const taskItem = createTaskItem(taskText); //DOM 요소 생성
    
    taskList.appendChild(taskItem); // DOM 삽입
    
    taskInput.value = ''; // DOM 초기화
  }
});
```

Reply:
가독성을 위해서 조건문을 잘 다루는게 중요한데 이렇게 해보면 더 좋을 것 같아요.
`Early Return` 혹은 `Guard Clause`라고 방식이에요.

```suggestion
  const taskText = taskInput.value.trim();
  
  if (event.key !== 'Enter' || taskText === '') {
    return;
  }
  
  const taskItem = createTaskItem(taskText);
  taskList.appendChild(taskItem);
  taskInput.value = '';
});
```

Reply:
훨씬 가독성이 좋아요!

Reply:
Guard clause는 주로 _**함수의 시작 부분에서 입력 유효성을 확인**_하는 데 사용되고, 
Early Return은 함수 내에서 **_중간에 조건을 확인하고 실행을 중단_**하는 데 사용됩니다.